chrome/content/zotero/xpcom/style.js

```diff
@@ -81,9 +81,42 @@ Zotero.Styles = new function() {
 		
 		// hidden dir
 		dir.append("hidden");
-		if(dir.exists()) i += _readStylesFromDirectory(dir, true);
+		if (dir.exists()) i += _readStylesFromDirectory(dir, true);
+		
+		// Sort visible styles by title
+		_visibleStyles.sort(function(a, b) {
+			return a.title.localeCompare(b.title);
+		})
+		// .. and freeze, so they can be returned directly
+		_visibleStyles = Object.freeze(_visibleStyles);
 		
 		Zotero.debug("Cached "+i+" styles in "+((new Date()).getTime() - start)+" ms");
+		
+		// transfer and reset "export.bibliographyLocale" pref value
+		var bibliographyLocale = '';
+		bibliographyLocale = Zotero.Prefs.get("export.bibliographyLocale");
+		if (bibliographyLocale) {
```

Comment:
We might want to move this outside of init into some place that is executed when the file is loaded, since init is not called until it's needed and by that time some services may already be accessing the prefs. i.e. https://github.com/zotero/zotero/pull/623#issuecomment-111785054